packages/signature-controller/src/types.ts

```diff
@@ -71,11 +77,29 @@ export type MessageParamsTyped = MessageParams & {
         primaryType: string;
         message: Json;
       };
-
+  /** Signature method V1, V3 or V4 */
+  signatureMethod?: string;
```

Comment:
Where is this populated in the message params?

We're trying to move away from adding properties to the params that are more metadata and stored in the underlying `SignatureRequest`.

Reply:
I removed this